products/almalinux.md

```diff
@@ -41,12 +46,27 @@ releases:
 > [AlmaLinux OS](https://almalinux.org/) is an open source, community-owned and governed,
 > forever-free enterprise Linux distribution, focused on long-term stability, providing a robust
 > production-grade platform. AlmaLinux OS is binary-compatible with
-> [RHEL](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux), and is owned
-> and controlled by the non-profit AlmaLinux OS Foundation, managed by a community-elected
-> board of directors.
+> [RHEL](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux). It is owned
```

Comment:
It migh look better to remove `.  It` part. Also it will be nice if we remove one of `and` in this sentence.

Reply:
@usta I don't understand what you're suggesting here. Can you please use suggestions instead of comments? that will make it more clear what you're specifically looking for. https://haacked.com/archive/2019/06/03/suggested-changes/